### PR TITLE
Story/ngda 222 ios multi select modals

### PR DESCRIPTION
**Pull request by developertesting85**
Base modal setup
Multi-selection modal creation

### CODE REVIEW

File: MiXMultiAppPoC.FeatureApp1/src/MiXMultiAppPoC.FeatureApp1.iOS/Views/Main/CustomComponentsViewController.cs [hunk 1]
```diff
@@ -169,7 +219,114 @@ protected override void BindView()
...
+
+        private void OnCloseDialogInteractionRequested(object sender, EventArgs e)
+        {
+            if (_multiSelectionIsShowing)
+                HideMuliSelection();
+        }
+
+        #region Multi Select Modal
+
+        public void BindMultiSelectModal()
```
Comment (developertesting85) [inline MiXMultiAppPoC.FeatureApp1/src/MiXMultiAppPoC.FeatureApp1.iOS/Views/Main/CustomComponentsViewController.cs]: Creating a new binding set to the modal. Can this happen all at once or should it be separate as now?

File: MiXMultiAppPoC.FeatureApp1/src/MiXMultiAppPoC.FeatureApp1.iOS/Views/Main/CustomComponentsViewController.cs [hunk 2]
```diff
@@ -169,7 +219,114 @@ protected override void BindView()
...
+            bindingSet.Bind(this).For(v => v.OpenDialogInteraction).To(vm => vm.OpenDialogInteraction);
+            bindingSet.Bind(_modalMultiSelectionButton).To(vm => vm.ShowMultiSelectionCommand);
             bindingSet.Apply();
         }
+
+        #region Dialog
+
+        private void OpenDialogInteractionRequested(object sender, EventArgs e)
+        {
+            ShowMuliSelection();
```
Comment (developertesting85) [inline MiXMultiAppPoC.FeatureApp1/src/MiXMultiAppPoC.FeatureApp1.iOS/Views/Main/CustomComponentsViewController.cs]: Need a generic kind of way to do this, in order to be able to call a different modal on this same page.

File: MiXMultiAppPoC.Common/MiXMultiAppPoC.Common.iOS/Controls/SelfSizedTableView.cs [hunk 3]
```diff
@@ -0,0 +1,27 @@
+﻿using CoreGraphics;
+using UIKit;
+
+namespace MiXMultiAppPoC.Common.iOS.Controls
+{
+    public class SelfSizedTableView : UITableView
```
Comment (developertesting85) [inline MiXMultiAppPoC.Common/MiXMultiAppPoC.Common.iOS/Controls/SelfSizedTableView.cs]: Auto adjustment of the table

File: MiXMultiAppPoC.FeatureApp1/src/MiXMultiAppPoC.FeatureApp1.iOS/Views/Main/CustomComponentsViewController.cs (hunk 1 above)
Comment (EclipseZA) [reply MiXMultiAppPoC.FeatureApp1/src/MiXMultiAppPoC.FeatureApp1.iOS/Views/Main/CustomComponentsViewController.cs]: That's fine

**Pull request merged**